### PR TITLE
Fixes #1364: Updated color scheme for pop up info notifications

### LINKED ISSUE #1364: Try and suggest different color schemes for pop up info notifications
**Description:** The color scheme must be in accordance with our new UI. Student should try different colours with notification toast so as to match the UI.

### PR DESCRIPTION
**Pull request by harshit98**
Fixes #1364 

- Updated the color scheme for pop-up notifications.
- Made slight changes in the colors to look more-like material design.

Screenshots:

screen shot 2017-12-18 at 7 02 28 am
screen shot 2017-12-18 at 7 02 44 am
screen shot 2017-12-18 at 7 02 55 am

### CODE REVIEW

**Comment by RishabhJain2018** [comment]
@aka-jain @spyshiv  Please take a look at this !

File: frontend/src/css/modules/notification.scss [hunk 1]
```diff
@@ -11,36 +11,52 @@
...
-        // color: #ffffff
+       // color: #ffffff;
     }
 }
 
 .toast {
     word-break: initial;
     &.toast-success {
-        //     background-color: #51a351;
+        background-color: #74DB70;
```
Comment (aka-jain) [inline frontend/src/css/modules/notification.scss]: Pass these colors to variable.scss

File: frontend/src/css/modules/notification.scss [hunk 2]
```diff
@@ -11,36 +11,52 @@
...
-        // color: #ffffff;
+        color: #ffffff;
         &:hover {
             // color: #cccccc;
             text-decoration: none;
         }
     }
     label {
-        // color: #ffffff
+       // color: #ffffff;
```
Comment (aka-jain) [inline frontend/src/css/modules/notification.scss]: Remove this commented line please @harshit98

**Comment by aka-jain** [comment]
One change, create variables for these colors please.

**Comment by aka-jain** [comment]
Any update here @harshit98?

**Comment by gautamjajoo** [comment]
@aka-jain Can I work on it?

**Comment by RishabhJain2018** [comment]
Since @harshit98 seems to be inactive for a long time and @gautamjajoo have already opened a PR(#1472) by taking into considering the reviews of this PR. hence, I am closing this one.

**Pull request closed without merging**